### PR TITLE
e2e test support: Store test pod logs and events

### PR DESCRIPTION
**Pull request by sutaakar**
# Issue link

Fixes https://github.com/project-codeflare/codeflare-operator/issues/290

# What changes have been made

Store logs of all containers in test namespace. Also store all namespace events.

# Verification steps

See PR check logs.

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: test/support/core.go [hunk 1]
```diff
@@ -57,3 +57,35 @@ func GetPodLogs(t Test, pod *corev1.Pod, options corev1.PodLogOptions) []byte {
 
 	return bytes
 }
+
+func StoreAllPodLogs(t Test, namespace *corev1.Namespace) {
```
Comment (astefanutti) [inline test/support/core.go:61]: `StoreAllPodLogs` -> `storeAllPodLogs`?

File: test/support/events.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	eventLastSeenKey,
+	eventFirstSeenKey,
+	eventNameKey,
+	eventSubObjectKey,
+	eventTypeKey,
+	eventReasonKey,
+	eventMessageKey,
+}
+
+func StoreEvents(t Test, namespace *corev1.Namespace) {
```
Comment (astefanutti) [inline test/support/events.go:54]: `StoreEvents` -> `storeEvents`

File: test/support/core.go (hunk 1 above)
Comment (sutaakar) [reply test/support/core.go]: done

File: test/support/events.go (hunk 2 above)
Comment (sutaakar) [reply test/support/events.go]: done

**Pull request merged**